packages/core/src/baseclient.ts

```diff
@@ -347,7 +347,9 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
 
     const { beforeSend, sampleRate } = this.getOptions();
 
-    if (typeof sampleRate === 'number' && sampleRate > Math.random()) {
+    // 1.0 === 100% events are sent
+    // 0.0 === 0% events are sent
+    if (typeof sampleRate === 'number' && Math.random() > sampleRate) {
```

Comment:
As far as i know Math.random() is not a valid random generator
i think better to use some kind of the https://github.com/ckknight/random-js

Reply:
Sure, it's not security-grade randomness, but it'll do just fine for sampling :)